procfs-core/src/netstat.rs

```diff
@@ -0,0 +1,625 @@
+use crate::ProcResult;
+#[cfg(feature = "serde1")]
+use serde::{Deserialize, Serialize};
+use std::io::BufRead;
+
+/// Represents the data from `/proc/net/netstat`.
+#[derive(Debug, Clone)]
+#[cfg_attr(feature = "serde1", derive(Serialize, Deserialize))]
+pub struct Netstat {
```

Comment:
Do you think we should mark this struct as `#[non_exhaustive]`, in case new fields get added in newer kernels?

Reply:
yeah, good idea! I'll add `#[non_exhaustive]`